### PR TITLE
update default tx fee for skills according to dorado testnet

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

DELETE INCLUSIVE THIS AND BELOW FOR STANDARD PR
------

## Release summary

Version number: [e.g. 1.0.1]

## Release details

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/master/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] Lint and unit tests pass locally and in CI
- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
- [ ] I have regenerated the latest API docs
- [ ] I built the documentation and updated it with the latest changes
- [ ] I have added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](https://github.com/fetchai/agents-aea/blob/master/develop-image/README.md#publish) 
      and [`aea-user`](https://github.com/fetchai/agents-aea/blob/master/develop-image/user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

## Further comments

Write here any other comment about the release, if any.

### CODE REVIEW

**Review by jrriehl — COMMENTED** [review]
I noticed that the TAC integration test is failing. Also, this version discrepancy looks larger than I would expect: https://github.com/fetchai/agents-aea/runs/7071468945?check_suite_focus=true#step:10:6493. Will that be resolved when the registry is updated? Otherwise, looks good.

**Comment by solarw** [comment]
> I noticed that the TAC integration test is failing. Also, this version discrepancy looks larger than I would expect: https://github.com/fetchai/agents-aea/runs/7071468945?check_suite_focus=true#step:10:6493. Will that be resolved when the registry is updated? Otherwise, looks good.

tac fails cause soef issue, as i see.
versions/packages will be fixex after the registry update that will go right after merging this PR
also simple oracle fix added (to develop directly by mistake, but merged into this branch)

File: tox.ini [hunk 1]
```diff
@@ -59,11 +59,10 @@ commands =
 	; so install current aea in a normal way
 	pip install {toxinidir}[all]
 	python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-ethereum
-    python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-ethereum
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-cosmos
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-ledger-fetchai
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-cli-ipfs
-    pytest -rfE --doctest-modules aea packages/fetchai/connections packages/fetchai/protocols packages/fetchai/skills tests/ --cov=aea --cov=packages/fetchai/connections --cov=packages/fetchai/contracts --cov=packages/fetchai/protocols --cov=packages/fetchai/skills --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc {posargs}
```
[the remark this answers is not in the document]
Comment (solarw) [reply tox.ini:66]: @5A11 this caused something like double import of some protobufs

looks like tests lookup mechanism of pytest do some tricks

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**